### ISSUE TITLE
Jet smart filter & Jet search compatibility 

### ISSUE THREAD

**Issue by Nima786** [CrocoBlock/suggestions#5591]
Hi there, I use your products in different projects and happy to use them on our new stack ( using alongside Bricks Builder ). This excellent page page builder is getting more and more popular and some third party products and add-ons already started to show their interests. It would be great if we see some kind of compatibility of your products with Bricks builder as well. At least you can provide short code for the mentioned products to be able to use alongside the Bricks builder.

Thanks

**Comment by ghost** [comment]
I totally agree, that would be great.

**Comment by infobeaedu** [comment]
Also I agree. It would be excellent.

**Comment by Saultejera** [comment]
I agree, Bricks is becoming my main builder, I doubt I will use Elementor again.  That's why I miss Crocoblock in Bricks, it would be the perfect combination.

**Comment by yannicklmb** [comment]
No doubt about it. JetSmartFilters is the best solution for filtering content (it is also compatible with Gutenberg). I think it would be the perfect companion to Bricks.

**Comment by Michaelgimii** [comment]
+1, It would be great to be able to create a shortcode for each filter

**Comment by PieterBrink** [comment]
+1

**Comment by Floydtm** [comment]
At the moment we have done integration for jetengine and jetsmartfilters plugins. Also, we continue to work in this direction, stay tuned for updates

**Comment by aalexandr21** [comment]
was added
